### PR TITLE
Track YouTube PFP events

### PR DESCRIPTION
**Pull request by faresite**
## What does this change?
Implements YouTube PFP `adStart` and `adEnd` tracking. [trello card](https://trello.com/c/K9WRhLQV)
Both events are tracked within the `video preroll` event action and are distinguished by the metric number:
- `metric17` for adStart
- `metric18` for adEnd

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by buck06191 — APPROVED** [review]
lgtm 👍

**Review by gtrufitt — APPROVED** [review]
🆒

File: static/src/javascripts/projects/common/modules/atoms/youtube-player.js [hunk 1]
```diff
@@ -194,13 +199,33 @@ export const initYoutubePlayer = (
             console.dir(event);
         };
 
+        const gaTracker = config.get('googleAnalytics.trackers.editorial');
+
+        const onAdStart = () => {
```
Comment (GHaberis) [inline static/src/javascripts/projects/common/modules/atoms/youtube-player.js]: Could we add a `void` return type to `onAdStart`/`onAdEnd`?

File: static/src/javascripts/projects/common/modules/video/ga-helper.js [hunk 2]
```diff
@@ -43,5 +43,23 @@ const getGoogleAnalyticsEventAction = (mediaEvent: MediaEvent) => {
     return action;
 };
 
+type PfpEventType = 'adStart' | 'adEnd';
+
+const buildPfpEvent = (pfpEventType: PfpEventType, videoId: string) => {
```
Comment (GHaberis) [inline static/src/javascripts/projects/common/modules/video/ga-helper.js]: Could we add a return type to `buildPfpEvent `?

**Pull request merged**